backend/de.metas.async/src/main/java/de/metas/async/Async_Constants.java

```diff
@@ -45,7 +45,7 @@ public final class Async_Constants
 	 */
 	public static final int DEFAULT_RETRY_TIMEOUT_MILLIS = 5000;
 
-	public static final String C_Async_Batch = "C_Async_Batch";
+	public static final String AsyncBatchId = "AsyncBatchId";
```

Comment:
consider prefixing it with `DYNATTR_` or moving it to the class where is used, if it's used only in one place....

Reply:
consider making this private and introduce 2 methods for setting/getting the dynamic attribute.
In this way you can make sure u are setting/getting the same datatype.